server/models/Volunteer.ts

```diff
@@ -0,0 +1,33 @@
+import { model, models, Model, Schema, Document } from "mongoose";
+import { Event, Volunteer } from "utils/types";
+
+export const VolunteerSchema = new Schema({
+    email: {
+        type: String,
+        required: true,
+    },
+    name: {
+        type: String,
+        required: true,
+    },
+    phone: {
+        type: String,
+        required: false,
+    },
+    filledForm: {
+        type: Boolean,
+        required: true,
+    },
+    attendedEvents: {
+        type: [Event],
```

Comment:
I noticed some tests are failing and say that `Event` is not a type. You might need to import and use `EventSchema`. One type is for Typescript and the other is used with the db.

Reply:
Thank you! I was wondering why it wasn't recognizing the Event type, but that makes sense. I'll change to EventSchema and see if it fixes the issue.

Reply:
Changed it, but still having some issues. Would it be better to store Event identifiers as an array of strings instead of an array of the actual event? I guess I'm not sure how the db handles nested schemas.

Reply:
I'm not sure how we want to handle the references tbh. I don't think strings will work since there is no good way of representing an object as a string and vice verca. I think the best way will be to store an array of ObjectIDs that Mongodb can reference, or we can denormalize the event data that a volunteer needs. Not too sure, let me know if anyone has thoughts on this. For now, having an array of type Event will denormalize the data which is fine.

Reply:
Once I reread what you wrote, I think you meant a string of ObjectIDs. That's one way of doing it, or we can store ObjectIDs and have mongodb auto-populate the referenced document: https://mongoosejs.com/docs/populate.html.